pkg/kubelet/probe.go

```diff
@@ -66,10 +66,17 @@ func (kl *Kubelet) probeContainer(pod *api.Pod, status api.PodStatus, container
 	ref, ok := kl.containerRefManager.GetRef(containerID)
 	if !ok {
 		glog.Warningf("No ref for pod '%v' - '%v'", containerID, container.Name)
-	} else {
+		return live, err
```

Comment:
Because of "if" instructions in lins 44 and 49 we know that at this point live is equal to probe.Success
It would be much more readable to change it to:
  "return probe.Success, err"
here